### DISCUSSION TITLE
Dar zoom como em outras IDEs

### DISCUSSION THREAD

**Discussion by luisfbp-hub** [dgadelha/Portugol-Webstudio#315] [Ideias e sugestões]
Seria bom se pudesse dar zoom, durante a progamação... Como no VScode

**Comment by dngadelha** [comment]
Olá, @luisfbp-hub!

Adicionamos uma opção na tela de configurações para alterar o tamanho da fonte do editor.

![image](https://github.com/user-attachments/assets/b0ebbfda-6653-4a6f-a5a5-958014f55ac3)

Essa opção já está disponível em https://portugol.dev/.

Lembrando que se você quiser alterar o tamanho da IDE toda, você ainda pode utilizar `CTRL +` ou `CTRL -`.
